geant4.spec

```diff
@@ -49,7 +49,6 @@ cmake ../%{n}.%{realversion} \
   -DGEANT4_BUILD_TLS_MODEL:STRING="global-dynamic" \
   -DGEANT4_ENABLE_TESTING=OFF \
   -DGEANT4_BUILD_VERBOSE_CODE=OFF \
-  -DGEANT4_BUILD_BUILTIN_BACKTRACE=OFF \
```

Comment:
this line better do not remove - Geant4 native backtrace may mess up CMS exceptions.